### ISSUE TITLE
External display output no longer works on MUXed laptop

### ISSUE THREAD

**Issue by korewaChino** [T-vK/MobilePassThrough#12]
I am running Fedora 34 and ran your script by renaming the F30 scripts to 34 and it works fine, although GDM no longer detects my external display.
Specs:
- HP OMEN 15 ce-020tx (2018)
- NVIDIA GeForce GTX1050 
- Intel Core i7-7700HQ (w/ Intel UHD630)

**Comment by T-vK** [comment]
I have made a lot of changes to the scripts in the last couple of weeks. (Currently only available in the [amd-gpu branch](https://github.com/T-vK/MobilePassThrough/tree/amd-gpu).) Those changes include a lot of bugfixes and improvements. I just haven't merged those changes into the master branch yet because I haven't tested them with Nvidia notebooks yet. 
But it might just work if you wanna give it a try.

**Comment by korewaChino** [comment]
Still does not work, same error on optirun and Xserver.

**Comment by T-vK** [comment]
Can you test it with wayland and check if it that would work on your device?

**Comment by korewaChino** [comment]
on Wayland at the moment, 
does not work either. NVIDIA driver straight up refuses to load.

**Comment by korewaChino** [comment]
Okay, I did a clean reinstall and just installed the NVIDIA driver by itself, it worked

**Comment by T-vK** [comment]
Yeah, the script is pretty "dumb" at the moment. It doesn't check what's installed already and just tries to install the nvidia driver along with bumblebee. The whole project really has only been tested on a fresh installation of Fedora.

**Comment by korewaChino** [comment]
I seem to have nailed down the issue to PRIME and Bumblebee. Multi-monitor setups do not work with this current configuration for some reason.

**Comment by T-vK** [comment]
Nice! That's awesome!
Have you found a solution to make bumblebee work with your setup?

**Comment by korewaChino** [comment]
Nope. Seems to be a MUX issue actually. Turns out my system is not actually MUXed but your script says so. The AMD-GPU fork does say it's not MUXed though. Anyway, you're welcome. I wanted this script to work on Ubuntu anyways.

**Comment by korewaChino** [comment]
aanyway might be offtopic but https://github.com/T-vK/MobilePassThrough/pull/13

**Comment by T-vK** [comment]
Yeah, detecting if and how the device is MUXed is tricky and not always possible. The most accurate way of telling that I'm aware of so far is checking as hat kind of devices the GPUs show up in lspci. 

Sorry for all the hassle btw. The project is very hard to maintain because I only have access to a very limited amount of devices at a time.

I made a bunch of changes to the project today after accepting your PR btw. Merging some new features I've been working on the last couple of days and also cleaning the project up by restructuring it a bit. These changes won't fix your problems, but nevertheless I was wondering if you'd like to test them because the project is currently in a state where I think it might make sense to have a first beta release.
